File: 2037-minimum-number-of-moves-to-seat-everyone/main.go
func minMovesToSeat(seats []int, students []int) int {
    sort.Ints(seats)
    sort.Ints(students)

    var totalMoves int = 0
    m := len(seats)
    for i := range(m) {
        totalMoves += abs(seats[i] - students[i])
    }

    return totalMoves
}

func abs(x int) int {
    if x < 0 {
        return -x
    }
    return x
}

File: 2037-minimum-number-of-moves-to-seat-everyone/main.py
class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
        
        seats.sort()
        students.sort()

        total_moves = 0

        m = len(seats)
        for i in range(m):
            total_moves += abs(seats[i] - students[i]) 
        
        return total_moves
